controllers/akodeploymentconfig/akodeploymentconfig_controller_avi_phase.go

```diff
@@ -27,15 +29,44 @@ import (
 	akov1alpha1 "github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pkg/apis/ako/v1alpha1"
 )
 
+func getAviCAFromADC(c client.Client, ctx context.Context,
+	log logr.Logger, obj *akoov1alpha1.AKODeploymentConfig) (string, error) {
+	aviControllerCA := &corev1.Secret{}
+	if err := c.Get(ctx, client.ObjectKey{
```

Comment:
duplicate function here https://github.com/vmware-samples/load-balancer-operator-for-kubernetes/blob/f81e1859dbace776a770598f330f019ddab66c31/controllers/akodeploymentconfig/user/user_controller.go#L252 

consider refactor?